driver-kotlin-sync/src/test/kotlin/com/mongodb/kotlin/client/MongoClientTest.kt

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.mongodb.kotlin.client
+
+import com.mongodb.ClientSessionOptions
+import com.mongodb.client.MongoClient as JMongoClient
+import kotlin.reflect.full.declaredFunctions
+import kotlin.test.assertEquals
+import org.bson.BsonDocument
+import org.bson.Document
+import org.junit.jupiter.api.Test
+import org.mockito.Mock
+import org.mockito.kotlin.any
+import org.mockito.kotlin.doReturn
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.refEq
+import org.mockito.kotlin.times
+import org.mockito.kotlin.verify
+import org.mockito.kotlin.verifyNoMoreInteractions
+import org.mockito.kotlin.whenever
+
+class MongoClientTest {
+
+    @Mock val wrapped: JMongoClient = mock()
+    @Mock val clientSession: ClientSession = ClientSession(mock())
+
+    @Test
+    fun shouldHaveTheSameMethods() {
+        val jMongoClientFunctions = JMongoClient::class.declaredFunctions.map { it.name }.toSet()
+        val kMongoClientFunctions = MongoClient::class.declaredFunctions.map { it.name }.toSet()
```

Comment:
Why not do the same `.filterNot { it.name == "wrapped" }` etc. check here (and elsewhere), as I've seen in some other `shouldHaveTheSameMethods` methods?

Reply:
Due to the lack of properties in the kotlin class.